src/pages/Location.tsx

```diff
@@ -650,12 +650,13 @@ const Location: React.FC<LocationProps> = ({ storageLocation, locationSearchPara
           )}
         </div>
       </AppShell.Main>
-      <PromptOnLeave
-        when={awaitingLabwares.length > 0}
-        messageHandler={awaitingStorageCheckOnExit}
-        message={'You have labwares that are not stored. Are you sure you want to leave?'}
-        onPromptLeave={onLeave}
-      />
+      {awaitingLabwares.length > 0 && (
+        <PromptOnLeave
+          when={awaitingStorageCheckOnExit}
+          message={'You have labwares that are not stored. Are you sure you want to leave?'}
```

Comment:
Labware is a mass noun. Can this be changed to "You have unstored labware." please?

Reply:
👍